src/containers/AccountConnection.jsx

```diff
@@ -370,6 +376,15 @@ class AccountConnection extends Component {
       success || queued ? this.buildSuccessMessages(konnector) : []
     return (
       <div className={styles['col-account-connection']}>
+        {!!konnector.available_version && (
+          <ButtonAction
```

Comment:
This button should be busy/disabled while redirecting, as it is an asynchronous call.